mautic/mautic
# Utm tags

Please answer the following questions:

| Q             | A
| ------------- | ---
| Bug fix?      | n
| New feature?  | y
| BC breaks?    | n
| Deprecations? | n
| Fixed issues  |  n

## Description
This PR 
-Adds a new form action to save UTM tags when submitting a form, utm tags are saved from referer URL
-Saves UTM tags from URL in landing pages
-Displays query string parameters and UTM tags in contacts history, if saved from a form it displays the form id from form submitted

## Steps to test this PR
Download PR:
1. create a new form with a save utm tags action, when the form is submitted it should save UTM tags to contact if the url parameters are passed on to the form.
2. create a new page, it should save UTM tags that would come in the url to view the landing page.

UTM tags that are being recorded:
utm_campaign
utm_content
utm_medium
utm_source
utm_term

3. When you view contact history you will see UTM tag history in the timeline.
